projects/@shared/components/collapsible-list.vue

```diff
@@ -172,17 +166,22 @@ export default {
     background-color: var(--selection-bg-color);
   }
 
-  .collapsible-list__section summary > svg {
+  .collapsible-list__section summary > .app-icon {
     position: absolute;
     top: 50%;
     left: 0;
     transform: translateY(-50%);
     transition: transform 0.15s ease-out;
+    width: 12px;
```

Comment:
The 12px width and height is a repetitive value for the icon. Maybe expand the icon component with a size prop and pass in t-shirt size format.